src/pkg/event/asgterminate/v2/awsevent.go

```diff
@@ -23,36 +23,36 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
-// AwsEvent contains the properties defined in AWS EventBridge schema
+// AWSEvent contains the properties defined in AWS EventBridge schema
```

Comment:
I chose not to use that approach because it spreads the logic needed to support an event across multiple packages, which then may need to be updated if new event types are added or removed.

This approach places all of the logic specific to an event type in a single, dedicated package. Adding or removing support for a new event type is accomplished by simply adding or removing its parser to/from the aggregated list.

The rest of the app interacts with these events only through the [`Event` interface](https://github.com/aws/aws-node-termination-handler/pull/612/files#diff-03278e672080d5c6852e33bfdee29f67f775d3394fbba5636e64dda0627b1eeaR49-R54), so there are no event type checks to handle special-cases for certain events.